releasenotes/notes/0.18/add-alignment-management-passes-650b8172e1426a73.yaml

```diff
@@ -9,11 +9,10 @@ features:
     restrictions on time allocation of instructions for a backend.
 
     If a backend has a restriction on the alignment of
-    :class:`~qiskit.circuit.Measure` instructions (in terms of quantization in time), the
+    instructions (in terms of quantization in time), the
     :class:`~qiskit.transpiler.passes.AlignMeasures` pass is used to adjust
-    delays in a scheduled circuit to ensure that the
-    :class:`~qiskit.circuit.Measure` instructions in the circuit are aligned
-    given the constraints of the backend. The
+    delays in a scheduled circuit to ensure that instructions in the circuit
+    are aligned given the constraints of the backend. The
```

Comment:
Was this due to #6705 ?

Reply:
It was, but it was a mistake on my part. I've fixed it in https://github.com/Qiskit/qiskit-terra/pull/6652/commits/6155b6ad7f4e12f43f27733a4526739b293a35ba